src/main/java/Date.java

```diff
@@ -0,0 +1,62 @@
+import java.lang.reflect.Array;
+
+public class Date {
+    private int day;
+    private int month;
+    private int year;
+
+    public Date(String inputDate) {
+        String[] tempStringArr = inputDate.split("/");
+        this.day = Integer.parseInt((String)Array.get(tempStringArr, 0));
+        this.month = Integer.parseInt((String)Array.get(tempStringArr, 1));
+        this.year = Integer.parseInt((String)Array.get(tempStringArr, 2));
+    }
+
+    public String monthAsString() {
```

Comment:
Can consider a hashmap implementation of <Integer, String> key-value pair to reduce code length. Else this works too!